# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/neofetch/t2-logo.patch
# Copyright (C) 2022 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- neofetch-7.1.0/neofetch.orig	2022-09-22 22:24:44.999130182 +0200
+++ neofetch-7.1.0/neofetch	2022-09-22 23:16:41.811332569 +0200
@@ -9823,15 +9823,23 @@
         ;;
 
         "t2"*)
-            set_colors 7 4
+            set_colors 1 7 3 4
             read -rd '' ascii_data <<'EOF'
-${c2}
-TTTTTTTTTT
-    tt   ${c1}222${c2}
-    tt  ${c1}2   2${c2}
-    tt     ${c1}2${c2}
-    tt    ${c1}2${c2}
-    tt  ${c1}22222${c2}
+${c3}
+                        //${c2}
+ ${c1}TTTTTTTTTTTTTTT${c2}       ${c3}//${c2}
+ ${c1}TTTTTTTTTTTTTTT${c2}      ${c3}//${c2} ${c1}..::${c2}  ${c4}:-:.${c2}
+       ${c4}ttt${c2}           ${c3}//${c2} ${c1}.::.${c2}    ${c4}.:-:${c2}
+       ${c4}ttt${c2}          ${c3}//${c2}  ${c1}.:.${c2}       ${c4}--${c2}
+       ${c4}ttt${c2}         ${c3}//${c2}            ${c4}:-:${c2}
+       ${c4}ttt${c2}        ${c3}//${c2}           ${c4}.:-:${c2}
+       ${c4}ttt${c2}       ${c3}//${c2}           ${c4}:-:.${c2}
+       ${c4}ttt${c2}      ${c3}//${c2}          ${c4}.--.${c2}
+       ${c4}ttt${c2}     ${c3}//${c2}         ${c4}.:-:${c2}
+       ${c4}ttt${c2}    ${c3}//${c2}          ${c4}...${c2}
+       ${c4}ttt${c2}   ${c3}//${c2}
+            ${c3}//${c2}         ${c3}.:::::::::::.${c2}
+           ${c3}//${c2}
 EOF
         ;;
 
